architecture/additional_concepts/scm.adoc

```diff
@@ -10,4 +10,15 @@
 toc::[]
 
 == Overview
-OpenShift source code repositories are referenced or owned by a build config and associated with projects.
\ No newline at end of file
+OpenShift takes advantage of preexisting source control management (SCM) systems
+either hosted internally (in-house git server) or externally (eg. https://github.com/[github],
+https://bitbucket.org/[bitbucket], etc.), although initially supporting only
+https://git-scm.com/[GIT] solutions.
```

Comment:
minor suggestion to use the proper capitalization of those words/brands: GitHub, Bitbucket and Git.